hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsBlobClient.java

```diff
@@ -435,6 +434,11 @@ public AbfsRestOperation createPath(final String path,
               null);
```

Comment:
These conditions can be combined as follows:
```
      if (op1Result != null) {
        boolean isDir = checkIsDir(op1Result);
        if (isFile == isDir) {
          throw new AbfsRestOperationException(HTTP_CONFLICT,
              AzureServiceErrorCode.PATH_CONFLICT.getErrorCode(),
              PATH_EXISTS,
              null);
        }
      }
```

Reply:
taken